README.md

```diff
@@ -56,6 +56,12 @@
 ### Key Features <a name="key-features"></a>
 - **Set up webpack and linter config**
 - **Add markup and basic styles**
+- **Create a new game on page load**
+- **Display all socres in the leaderboard**
+- **Create new score**
+- **Create new score**
+- **Get all scores**
+- **store game id and all scores in local storage**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
```

Comment:
- [ ] Kindly make sure you update your `Getting Started` section with some important information such as on how to `install` your appl, `access` the cloned directory, `to run your application`,... To help you just refer this **[example](https://gist.github.com/tresorsawasawa/cc5e9a9e5dd20c0179ae98eca3b1ae96)**. By adding those information, it will help non technical users on how to use your application